sdk/src/test/java/com/mapbox/maps/MapControllerTest.kt

```diff
@@ -108,14 +108,15 @@ class MapControllerTest {
 
   @Test
   fun onStop() {
+    mapController.onStart()
```

Comment:
why do we need `onStart()` here?

Reply:
Update the state to started stated

Reply:
the test looks pretty strange then
can we introduce new enum value `LifecycleState.STATE_INIT` instead of setting `lifecycleState` to `LifecycleState.STATE_STOPPED`? why is it stopped by default?

Reply:
A normal lifecycle should be onStart->onStop. We should not let onStop make any changes to the sdk before onStart is invoked firstly.

Reply:
Yeah, I guess it sounds logical.